docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+---
+title: 2021 MUI Developer Survey results
+description: Results for our yearly developer survey, 2021 edition.
+date: 2022-02-22T00:00:00.000Z
+authors: ['danilo-leal']
+tags: ['Developer survey']
+---
+
+Keeping up with the tradition, a few months ago we opened the 2021 MUI Developer Survey, to which we received 1,589 responses, 7% more answers than last year. This time, we have almost doubled the survey size, adding more depth to each covered topic, especially opening space for more open-ended questions. We wholeheartedly appreciate everyone who contributed, you are all helping shape the future of MUI as a company, so, thank you!
+
+Just like the previous iterations, the survey was chopped into three sections: ["Your needs"](#your-needs), ["Your product"](#your-product), and ["About you"](#about-you).
+
+## Your needs
+
+### How would you feel if you could no longer use MUI?
+
+Fortunately, and similar to 2019 and 2020, over 93% of the respondents would be disappointed if they could no longer use MUI (a sum of _"Very disappointed"_ with _“Somewhat disappointed”_). A very slight 1% decrease from 2020 (94%) but still as encouraging as it can be. MUI’s team will continue to grow to keep you all even more satisfied.
```

Comment:
Let's tighten this up to underscore the most important point: everybody loves us! 😁 I would also suggest replacing the third-person nouns with "you," and "MUI's team" with "we."

```suggestion
Over 93% of you would be disappointed if you could no longer use MUI (a sum of _"Very disappointed"_ and _“Somewhat disappointed”_). That's in keeping with 94% of you who felt the same way when we asked this question in 2019 and 2020. We'll continue to do our best to keep you satisfied over the long term!
```